LocalFeeder/Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM python:3.10.6-slim-bullseye
+
+RUN apt-get update 
+RUN apt-get install -y git ssh
+
+
+RUN mkdir LocalFeeder
+COPY  * ./LocalFeeder
+WORKDIR ./LocalFeeder
+
+RUN pip install -r requirements.txt
+
+EXPOSE 8765/tcp
```

Comment:
This docker file is the only one to have a 8765 instead of 8766. Why?

Reply:
This file now gets auto-generated port here does not matter